idl/user-leds.idol

```diff
@@ -0,0 +1,37 @@
+// User LED blinky API
+
+Interface(
+    name: "UserLeds",
+    ops: {
+        "led_on": (
+            args: {
+                "index": "usize",
+            },
+            reply: Result(
+                ok: "()",
+                err: CLike("LedError"),
+            ),
+            idempotent: true,
+        ),
+        "led_off": (
+            args: {
+                "index": "usize",
+            },
+            reply: Result(
+                ok: "()",
+                err: CLike("LedError"),
+            ),
+            idempotent: true,
+        ),
+        "led_toggle": (
+            args: {
+                "index": "usize",
+            },
+            reply: Result(
+                ok: "()",
+                err: CLike("LedError"),
+            ),
+            idempotent: true,
```

Comment:
Is that one idempotent? I think not. It would toggle more or fewer times.

Reply:
Good catch!

Reply:
Fixed in latest patchset.